app/integrations/comp_and_pen/scheduled_message_helpers.py

```diff
@@ -0,0 +1,207 @@
+import boto3
+from boto3.dynamodb.conditions import Attr
+from flask import current_app
+
+from app.models import (
+    SMS_TYPE,
+    Service,
+    Template,
+)
+from app.notifications.send_notifications import send_notification_bypass_route
+from app.va.identifier import IdentifierType
+
+
+class CompPenMsgHelper:
+    dynamodb_table = None
+
+    def __init__(self, dynamodb_table_name: str) -> None:
+        """
+        This class is a collection of helper methods to facilitate the delivery of schedule Comp and Pen notifications.
+
+        :param dynamodb_table_name (str): the name of the dynamodb table for the db operations, required
+        """
+        self.dynamodb_table_name = dynamodb_table_name
+
+    def _connect_to_dynamodb(self, dynamodb_table_name: str = None) -> None:
+        """Establishes a connection to the dynamodb table with the given name.
+
+        :param dynamodb_table_name (str): the name of the dynamodb table to establish a connection with
+
+        Raises:
+            ClientError: if it has trouble connectingto the dynamodb
+        """
+        if dynamodb_table_name is None:
+            dynamodb_table_name = self.dynamodb_table_name
+
+        # connect to dynamodb table
+        dynamodb_resource = boto3.resource('dynamodb')
+        self.dynamodb_table = dynamodb_resource.Table(dynamodb_table_name)
+
+    def get_dynamodb_comp_pen_messages(
+        self,
+        message_limit: int,
+    ) -> list:
+        """
+        Helper function to get the Comp and Pen data from our dynamodb cache table.
+
+        Items should be returned if all of these attribute conditions are met:
+            1) item exists on the `is-processed-index`
```

Comment:
Can we get rid of this logic `payment_id is not equal to -1 (placeholder value)`

Reply:
updated